cypress/support/commands.js

```diff
@@ -32,10 +32,31 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+Cypress.Commands.add('register', ({ email, username, password }) => {
   cy.request('POST', '/users', {
```

Comment:
Ensure that the endpoint '/users' is correct for user registration. If the API endpoint is different, update it accordingly.